client/src/elm/Pages/AcuteIllnessProgressReport/View.elm

```diff
@@ -12,7 +12,7 @@ import Backend.Person.Utils exposing (ageInMonths, ageInYears, isChildUnderAgeOf
 import Date
 import EverySet exposing (EverySet)
 import Gizra.Html exposing (emptyNode)
-import Gizra.NominalDate exposing (NominalDate, diffDays, diffMonths, formatDDMMYY)
+import Gizra.NominalDate exposing (NominalDate, diffDays, diffMonths, formatDDMMyyyy)
```

Comment:
Please add screenshots for changes at this file.
Since date becomes longer, we may need to adjust the styling.

Reply:
![Screen Shot 2021-12-01 at 15 10 27](https://user-images.githubusercontent.com/32199791/144240296-9364a0ec-190a-4745-a412-a37023c32b54.png)